ehsm_kms_service/k8s/ehsm-kms.yaml

```diff
@@ -178,6 +187,131 @@ spec:
+        image: intel/ehsm_kms_service:latest
+        imagePullPolicy: IfNotPresent
+        readinessProbe:
+          exec:
+            command:
+            - sh
+            - -c
+            - "curl http://${EHSM_CONFIG_COUCHDB_SERVER}:${EHSM_CONFIG_COUCHDB_PORT}/"
+          initialDelaySeconds: 10
+          periodSeconds: 10
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /dev/sgx/enclave
+          name: dev-enclave
+        - mountPath: /dev/sgx/provision
+          name: dev-provision
+        - mountPath: /var/run/aesmd
+          name: dev-aesmd
+        - mountPath: /var/run/ehsm
+          name: dev-dkeyprovision
+        env:
+        - name: PCCS_URL
+          valueFrom:
+            configMapKeyRef:
+              name: ehsm-configmap
+              key: pccs_url
+        - name: EHSM_CONFIG_COUCHDB_USERNAME
+          valueFrom:
+            secretKeyRef:
+              name: ehsm-secret
+              key: couch_root_username
+        - name: EHSM_CONFIG_COUCHDB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: ehsm-secret
+              key: couch_root_password
+        - name: EHSM_CONFIG_COUCHDB_SERVER
+          valueFrom:
+            configMapKeyRef:
+              name: ehsm-configmap
+              key: database_url
+        - name: EHSM_CONFIG_COUCHDB_PORT
+          valueFrom:
+            configMapKeyRef:
+              name: ehsm-configmap
+              key: database_port
+        - name: EHSM_CONFIG_COUCHDB_DB
+          valueFrom:
+            configMapKeyRef:
+              name: ehsm-configmap
+              key: database_name
+        - name: EHSM_CONFIG_DENY_SIZE_LIMIT
+          valueFrom:
+            configMapKeyRef:
+              name: ehsm-configmap
+              key: deny_size_limit
+        - name: EHSM_CONFIG_DENY_TIMEOUT
+          valueFrom:
+            configMapKeyRef:
+              name: ehsm-configmap
+              key: deny_timeout
+        - name: EHSM_CONFIG_DENY_COUNT_LIMIT
+          valueFrom:
+            configMapKeyRef:
+              name: ehsm-configmap
+              key: deny_count_limit
+        ports:
+        - name: ehsm-acs
+          containerPort: 9010
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: ehsm-acs-service
+  namespace: ehsm-kms
+spec:
+  type: LoadBalancer
+  selector:
+    app: ehsm-acs
+  ports:
+    - name: ehsm-acs
+      protocol: TCP
+      # This port is [ehsm access control service] access port,you can change it to what you want.
+      port: 9010
+      targetPort: 9010
+      nodePort: 30010
+  sessionAffinity: ClientIP
+  externalIPs:
+  # This IP is [ehsm access control service] access IP, You need to adjust 5.6.7.8 to the service IP you want to expose.
+  - 5.6.7.8
+
```

Comment:
there's no difference with the ehsm-kms service? does it means the external user could also access it?

Reply:
It can be accessed in the LAN